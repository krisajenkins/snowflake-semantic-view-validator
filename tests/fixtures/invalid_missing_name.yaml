description: This is missing a name field
tables:
  - name: TEST_TABLE
    base_table:
      database: TEST_DB
      schema: PUBLIC
      table: TEST
    dimensions:
      - name: ID
        expr: ID
        data_type: VARCHAR
