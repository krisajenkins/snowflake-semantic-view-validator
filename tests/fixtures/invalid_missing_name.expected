════════════════════════════════════════════════════════════════════════════════
  VALIDATION ERROR
════════════════════════════════════════════════════════════════════════════════

✗ Failed to parse YAML file: missing field `name`

TIP:
  Check the YAML syntax at the indicated line and column.
  Common issues include:
    • Incorrect indentation (use spaces, not tabs)
    • Missing colons after keys
    • Unquoted strings containing special characters
    • Missing required fields

════════════════════════════════════════════════════════════════════════════════